earn/src/components/borrow/ManageAccountButtons.tsx

```diff
@@ -2,13 +2,21 @@ import { OutlinedWhiteButtonWithIcon } from 'shared/lib/components/common/Button
 
 import { ReactComponent as PlusIcon } from '../../assets/svg/plus.svg';
 
-export default function ManageAccountButtons() {
+export type ManageAccountButtonsProps = {
+  onAddCollateral: () => void;
+  onRemoveCollateral: () => void;
+  onBorrow: () => void;
+  onRepay: () => void;
+};
+
+export default function ManageAccountButtons(props: ManageAccountButtonsProps) {
```

Comment:
Since the modals are being defined in the parent component anyway, what's the point of defining the buttons here?

Reply:
To help reduce the size of the page component. Ideally, most of our components should be abstracted so that the page is concise. I could put the modals in here, but that would have its drawbacks, such as passing all of the data in here rather than just the listeners.